testcases/kernel/syscalls/flock/flock02.c

```diff
@@ -75,6 +75,7 @@
 #include <sys/types.h>
 #include <sys/file.h>
 #include <sys/wait.h>
+#include <fcntl.h>
 #include <errno.h>
 #include <stdio.h>
 #include "test.h"
```

Comment:
And can you please put the part  that adds fcntl.h to the flock tests into a separate patch please?

Otherwise it looks fine.